assets/scripts/progress.js

```diff
@@ -29,4 +29,6 @@ function numberWithCommas(x) {
   );
 
   document.getElementById("percent").textContent = `${percentage.toFixed(0)}%`
+
+  setTimeout(updatePage, 09/17/1787 * '📜'.charCodeAt(0) * 420);
```

Comment:
[09/17/1787](https://constitutioncenter.org/interactive-constitution/blog/today-the-constitution-was-signed-in-philadelphia)